versions.yaml

```diff
@@ -294,7 +294,8 @@ externals:
       package_output_dir: "OvmfX64"
     sev:
       description: "AmdSev build needed for SEV measured direct boot."
-      version: "edk2-stable202202"
+      url: "https://github.com/confidential-containers-demo/edk2"
```

Comment:
Or we manage to apply @dubek 's patch on the upstream edk2 sources on build time like we do for QEMU, for example.

In either case, @Alex-Carter01 , could you please add a `TODO` comment pointing out to the issue so that we remember that the change should be reverted once the updated edk2 is released?

Reply:
This PR switches the OVMF source because we uncovered a bug in the upstream OVMF SEV code. @dubek has submitted [this patch](https://edk2.groups.io/g/devel/message/97251) to edk2. So we're planning to switch back to upstream OVMF at the next OVMF release (February, I think). We are pointing to this repo only until then. Would Kata Containers be the right place for something temporary like that?

Reply:
@fitzthum just checking: that bug impacts online_sev_kbc only?

Reply:
offline and online. They both rely on secret injection and the efi_secret kernel module which takes the injected secrets and puts them on the filesystem. The bug basically breaks this kernel module so we can't use either KBS.

Reply:
https://github.com/kata-containers/edk2

Reply:
Great, thanks @fidencio !

@Alex-Carter01 , could you please update this PR to use that ^ tree?